<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VehicleTemplateSelectorWidget</class>
 <widget class="QWidget" name="VehicleTemplateSelectorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="2,0">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,4">
         <item>
          <widget class="QListWidget" name="templateList">
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOn</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGraphicsView" name="templateImage">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>250</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgba(254, 254, 254, 0);</string>
           </property>
           <property name="interactive">
            <bool>false</bool>
           </property>
           <property name="renderHints">
            <set>QPainter::Antialiasing|QPainter::HighQualityAntialiasing|QPainter::TextAntialiasing</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="templateDescription">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="undoRedoEnabled">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>Information about the Vehicle in short.</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
